### ISSUE TITLE
Add buddy.yml

### ISSUE THREAD

**Issue by mkornatz** [onedesign/generator-one-base#154]
Instead of having a tutorial on how to set up a Buddy pipeline manually, we should include all of those things in a standard buddy.yml file in our generator.

ref: https://github.com/onedesign/Guidelines/blob/master/deployment.md

**Comment by cmalven** [comment]
Should this add a new `buddy` sub-generator?

**Comment by mkornatz** [comment]
I don't think so, because the buddy pipelines will be specific to the framework and type of project.

**Comment by cmalven** [comment]
Right, but there will inevitably be occasions where a project doesn't use Buddy for deployment (especially if anyone outside of ODC uses this). I don't mind the generator being opinionated, but assuming the use of paid, 3rd-party services seems overly opinionated to me.
